update toml

emmakbat opened the pull request.

reed-foster commented on src/qnngds/cells.py:
```diff
@@ -334,9 +336,14 @@ def nanowires(
-        nanowire_ref = NANOWIRES << device.nanowire.spot(
-            channel_source_w[0], channel_source_w[1]
-        )
+        if lengths is None:
```
probably want to uncomment the else?

emmakbat replied:
Yes sorry -- before I had things set up correctly my code couldn't "see" the variable_lengths function so I was unable to test this part of the code, and I commented it out to be safe. Currently working on testing the else and adding a couple other improvements

reed-foster replied:
sounds good, let me know when the PR is ready

reed-foster commented:
I'm not sure why none of the github actions ran on this. Looks like it's cause this is from a fork, so you're considered a first-time contributor: https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks

I'll see if I can have it run the actions.

emmakbat commented:
Yeah, forking the repository was the recommended workflow before the refactoring. I'm not totally sure how to reset things on my machine so it's not working from the fork

reed-foster commented:
Seems like I just need to add `pull_request` event for the `on` section of the github action: https://stackoverflow.com/questions/73314842/unable-to-trigger-github-actions-when-pull-request-comes-from-a-forked-repositor

emmakbat commented:
Updated, and I also added a "tolerance" option to DieParameters that shrinks the pads to allow for MLA alignment error (which is inevitable, so accounting for it should be the default option)

reed-foster commented on docs/user/images/devices/resistor/meander_sc_contacts.png:
not sure why this changed lol

reed-foster commented on src/qnngds/cells.py:
```diff
@@ -302,6 +302,8 @@ def nanowires(
     device_layer: int = 1,
     text: Union[None, str] = None,
+    lengths: List[float] = None,
+    tolerance: float = 5,
```
should `tolerance` be in `die_parameters`?

reed-foster replied:
I see you already have `pad_tolerance` in `DieParameters`
